Libraries/Components/Button.windows.js

```diff
@@ -155,35 +155,46 @@ class Button extends React.Component {
     if (Platform.OS === "windows") {
       const tabIndex = this.props.tabIndex || 0;
       const windowsTabFocusable = !disabled && tabIndex >= 0;
-      content =
-        <FocusableWindows
-          ref={this._setFocusableRef}
+      return (
+        <Touchable
```

Comment:
>   <Touchable [](start = 5, length = 13)

But any reason for changing the embedding order (i.e. FW inside Touchable rather than what we had before)?

Reply:
I don't see how we changed order here. previous code produced Touchable>FW>View>Text and new one produces same tree.

---
In reply to: [158572207](https://github.com/reseul/react-native-windows/pull/1#discussion_r158572207) [](ancestors = 158572207)